src/plugins/intel_npu/tests/functional/behavior/remote_tensor_tests/dma_buf_remote_run.hpp

```diff
@@ -0,0 +1,260 @@
+    float* random_buffer_tensor = new float[byte_size / sizeof(float)];
+    memset(random_buffer_tensor, 1, byte_size);
+    ov::Tensor random_tensor_input{ov::element::f32, tensor.get_shape(), random_buffer_tensor};
+
+    OV_ASSERT_NO_THROW(inference_request.set_input_tensor(random_tensor_input));
+    OV_ASSERT_NO_THROW(inference_request.infer());
+
+    // set random output tensor
+    auto output_tensor = inference_request.get_output_tensor();
+    const auto output_byte_size = ov::element::get_memory_size(ov::element::f32, shape_size(output_tensor.get_shape()));
+
+    float* output_random_buffer_tensor = new float[output_byte_size / sizeof(float)];
+    memset(output_random_buffer_tensor, 1, output_byte_size);
+    ov::Tensor outputrandom_tensor_input{ov::element::f32, output_tensor.get_shape(), output_random_buffer_tensor};
+
+    OV_ASSERT_NO_THROW(inference_request.set_output_tensor(outputrandom_tensor_input));
+    OV_ASSERT_NO_THROW(inference_request.infer());
+
+    delete[] random_buffer_tensor;
+    close(fd_heap);
+}
+
+TEST_P(DmaBufRemoteRunTests, CheckOutputDataFromMultipleRuns) {
+    // Skip test according to plugin specific disabledTestPatterns() (if any)
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    ov::InferRequest inference_request;
+    float* data;
+
+    OV_ASSERT_NO_THROW(compiled_model = core->compile_model(ov_model, target_device, configuration));
+    OV_ASSERT_NO_THROW(inference_request = compiled_model.create_infer_request());
+    auto tensor = inference_request.get_input_tensor();
+
+    const auto byte_size = ov::element::get_memory_size(ov::element::f32, shape_size(tensor.get_shape()));
+
+    auto fd_heap = getFdDmaHeap(byte_size);
+
+    auto mmap_ret = mmap(NULL, byte_size, PROT_WRITE | PROT_READ, MAP_SHARED, fd_heap, 0);
+    if (mmap_ret == MAP_FAILED) {
+        ASSERT_FALSE(true) << "mmap failed.";
+    }
+
+    memset(mmap_ret, 1, byte_size);
+
+    auto context = core->get_default_context(target_device).as<ov::intel_npu::level_zero::ZeroContext>();
+
+    auto output_tensor = inference_request.get_output_tensor();
+    const auto output_byte_size = output_tensor.get_byte_size();
+    float* output_data_one = new float[output_byte_size / sizeof(float)];
+    ov::Tensor output_data_tensor_one{ov::element::f32, output_tensor.get_shape(), output_data_one};
+
+    {
+        auto remote_tensor = context.create_tensor(ov::element::f32, tensor.get_shape(), fd_heap);
+        OV_ASSERT_NO_THROW(inference_request.set_input_tensor(remote_tensor));
+        OV_ASSERT_NO_THROW(inference_request.set_output_tensor(output_data_tensor_one));
+        OV_ASSERT_NO_THROW(inference_request.infer());
+    }
+
+    float* output_data_two = new float[output_byte_size / sizeof(float)];
+    ov::Tensor output_data_tensor_two{ov::element::f32, output_tensor.get_shape(), output_data_two};
+
+    {
+        const size_t byte_size = tensor.get_byte_size();
+        data = new float[byte_size / sizeof(float)];
```

Comment:
I couldn't find a corresponding delete operator. Is it expected behavior?

Reply:
added, thanks!